Codecademy/gamut
# chore(Truncate)!: Remove `Truncate`

### Overview


Removes deprecated `Truncate` component.


### PR Checklist

- [x] Related to JIRA ticket: [WEB-1848]

#### Testing Instructions


Visit the alert story and ensure the general (non inline) alert gets truncated with the option to expand. Confirm that clicking the arrow expands the alert and un-truncates the text. Confirm the general inline alert does not get truncated. Confirm that no other alerts show the expand button.
See monolith and portal PRs for testing instructions.

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/31746)  | [Monolith Env](https://pr-31746-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/2281)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2281)   |
